files/messages/passive-discussion.txt

```diff
@@ -1 +1 @@
-I wanted to follow up on last night's instructor training discussion session - my understanding is that you didn't have questions, and didn't play an active part in the discussion.  These sessions are a key part of bringing new instructors into the community, so if you experienced any difficulties partcipating, please let us know so we can explore ways to be of more support. This could include the language the sessions are conducted in, the platform hosting the video calls, or other considerations.  Otherwise, I would be grateful if you could do another one some time in the next few weeks - if you don't have questions about our lessons or workshop operations, you can always bring thoughts or comments on other teaching or training you've done, and how it compares with our approaches.
+Thank you for participating in the instructor training discussion session on DD-MM. These sessions are a key part of bringing new instructors into the community and we thus monitor participants' involvement in them as part of your checkout procedure for instructor training. It is our understanding is that you didn't have questions, and didn't play an active part in the discussion. If you experienced any difficulties participating, please let us know so we can explore ways to be of more support. This could include the language the sessions are conducted in, the platform hosting the video calls, or other considerations.  Otherwise, we would be grateful if you could participate in another discussion session some time in the next few weeks - if you don't have questions about our lessons or workshop operations, you can always bring thoughts or comments on other teaching or training you've done, and how it compares with our approaches.
```

Comment:
Also, I think two negative statements in the following sentence can be upsetting (notice the extraneous **is**):
> It is our understanding **is** that you didn't have questions, and didn't play an active part in the discussion.

So, I suggest a 'weaker' statement:
> We find that due to some circumstances you were not able to actively participate in the discussion.